lib/ui/window/window.h

```diff
@@ -85,6 +99,10 @@ class Window final {
   WindowClient* client_;
   tonic::DartPersistentValue library_;
   ViewportMetrics viewport_metrics_;
+#if defined(OS_FUCHSIA)
```

Comment:
Since we are trying to remove Fuchsia specific code in the engine, this can be stored on the engine or platform view subclass in the Flutter runner on Fuchsia.

Reply:
Done

Reply:
I wasn't able to move this into PlatformView, but I did move it into SceneHost which is Fuchsia-specific